js/ui/windowManager.js

```diff
@@ -748,6 +748,8 @@ const WindowManager = new Lang.Class({
                                                        '/com/endlessm/Sylvester/Service');
         this._sylvesterListener.connectSignal('RotateBetweenPidWindows',
                                               Lang.bind(this, this._handleRotateBetweenPidWindows));
+        this._sylvesterListener.connectSignal('RevertXidRotation',
```

Comment:
Ah yes, I completely forgot about pushing that one. Here you go https://github.com/endlessm/sylvester/pull/5